java/ql/src/Security/CWE/CWE-940/AndroidIntentRedirection.qhelp

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+    <overview>
+        <p>An exported Android component that obtains a user-provided Intent and uses it to launch another component
+        can be exploited to obtain access to private, unexported components of the same app or to launch other apps' components
+        in behalf of the victim app.</p>
+    </overview>
+    <recommendation>
+        <p>Do not export compontents that start other components from a user-provided Intent. 
+        They can be made private by setting the `android:exported` property to `false` in the app's Android Manifest.</p>
```

Comment:
<code>android:exported</code> instead of backticks :)

Reply:
Ouch, good catch, thanks! Fixed in 4bf22f2.